hipchat.go

```diff
@@ -218,4 +225,5 @@ func (c *Client) listen() {
 			}
 		}
 	}
+	panic("unreachable")
```

Comment:
I don't understand why the `panic` was added. just in case something unexpected happens?